src/multiStore.js

```diff
@@ -2,27 +2,73 @@
  * Creates a multi-store
  *
  * @param {store[]} stores - The stores to sync.
+ * @param {object} [errorHandlers] - Functions to override the error handling behavior.
+ * @param {function} [errorHandlers.onGetError] - A function to execute if a store errors when getting a value.
```

Comment:
This may receive more options in the future, not only `errorHandlers`.

Same for next ones.

```suggestion
 * @param {function} [options.onGetError] - A callback to execute if a store errors when getting a value.
```